pep-0675.rst

```diff
@@ -747,13 +740,73 @@ others may consider it in the future, such extensions are out of scope
 for this PEP.
 
 
+Rejected Names
+--------------
+
+We considered a variety of names for the literal string type and
+solicited ideas on `typing-sig
+<https://mail.python.org/archives/list/typing-sig@python.org/thread/VB74EHNM4RODDFM64NEEEBJQVAUAWIAW/>`_.
+Some notable alternatives were:
+
++ ``Literal[str]``: This is a natural extension of the
+  ``Literal["foo"]`` type name, but typing-sig `objected
+  <https://mail.python.org/archives/list/typing-sig@python.org/message/2ZQO4NTJEI42KTRJDBL77MNANEXOW7UI/>`_
+  that users could mistake this for the literal type of the ``str``
+  class.
+
++ ``LiteralStr``: This is shorter than ``LiteralString`` but looks
+  weird to the PEP authors.
+
++ ``LiteralDerivedString``: This (along with
+  ``MadeFromLiteralString``) best captures the technical meaning of
+  the type. It represents not just the type of literal expressions
+  such as ``"foo"`` but also that of expressions composed from
```

Comment:
```suggestion
  the type. It represents not just the type of literal expressions,
  such as ``"foo"``, but also that of expressions composed from
```

See above:

> Actually, I made an oversight here that my sister spotted when giving me a second opinion on this—`such as ``"foo"`` ` should be set off by commas, rather than just having only a single comma after it. I'll make another suggestion to that effect.